### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by guess-the-riddle**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
...
+
+  test(`Should return zero, when Cat and Dog age = 0`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`Should return 2, when Cat and Dog age = 24`, () => {
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
```
Comment (murat-yesimbietov) [inline src/getHumanAge.test.js:29]: Add getHumanAge(23, 23) checking (boundary values)

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(29, 30))
+      .toEqual([3, 3]);
+  });
+
+  test(`Should return 4 and 3, when Cat and Dog age = 33`, () => {
+    expect(getHumanAge(33, 33))
+      .toEqual([4, 3]);
+  });
+
+  test(`Should return 4 and 3, when Cat and Dog age = 100`, () => {
```
Comment (murat-yesimbietov) [inline src/getHumanAge.test.js:41]: 4 and 3?

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]

**Review by murat-yesimbietov — APPROVED** [review]
